src/pybind/mgr/dashboard/frontend/src/app/core/navigation/administration/administration.component.html

```diff
@@ -1,7 +1,6 @@
 <cds-overflow-menu [customTrigger]="customTrigger"
                    [flip]="true">
-  <li class="cds--overflow-menu-options__option mb-2"
-      *ngIf="userPermission.read">
```

Comment:
why removing permission ?

Reply:
its actually being moved to a parent div.

Reply:
to here: https://github.com/ceph/ceph/pull/62149/files#diff-4f16ad27992de069c3f39c6e821edd7ea2cbade91164d2f254d1d3ae43f909bcR43

Reply:
and that fixes this empty icon
![image](https://github.com/user-attachments/assets/41666717-f452-4769-a204-891460e8c21f)